### PR TITLE
Add Twente Milieu integration

### PR DESCRIPTION
**Pull request by frenck**
**Description:**

Adds documentation for the Twente Milieu integration.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#25129

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/9851"></a>

### CODE REVIEW

File: source/_components/twentemilieu.markdown [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+title: "Twente Milieu"
+description: "Instructions on how to integrate Twente Milieu with Home Assistant."
+logo: twentemilieu.png
+ha_category:
+  - Sensor
+  - Environment
+ha_release: 0.97
+ha_iot_class: Cloud Polling
+ha_config_flow: true
```
Comment (MartinHjelmare) [inline source/_components/twentemilieu.markdown:10]: I had forgotten we had this key. Do we use it somewhere?

File: source/_components/twentemilieu.markdown [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+This integration provides sensors for the following waste pickup dates from Twente Milieu:
+
+- Next plastic waste pickup date.
+- Next organic waste pickup date.
+- Next paper waste pickup date.
+- Next non-recyclable waste pickup date.
+
+## Services
+
+The Twente Milieu integration exposes a service that allows you to manual update
```
Comment (MartinHjelmare) [inline source/_components/twentemilieu.markdown]:
```suggestion
The Twente Milieu integration exposes a service that allows you to manually update
```

File: source/_components/twentemilieu.markdown [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+the pick update from Twente Milieu.
```
Comment (MartinHjelmare) [inline source/_components/twentemilieu.markdown]:
```suggestion
the pickup date from Twente Milieu.
```

File: source/_components/twentemilieu.markdown (hunk 1 above)
Comment (frenck) [reply source/_components/twentemilieu.markdown:10]: It shows up in the sidebar on the integration page.
Currently working getting the set of properties into sync with the docs website, goal: Able to sync up with the code base.

After that, we can start adding more visual elements to the website (and hopefully filters and other stuff).

**Review by MartinHjelmare — APPROVED** [review]

**Comment by grote-beer** [comment]
Is there a specific reason as to why it is not possible to configure this via the configuration.yaml?

Still love this addition, was searching for exactly this only last week

**Comment by frenck** [comment]
This is not really the place to discuss these kinds of things. Please use the forum or our Discord chat for that.
Regarding your question: Nope, no specific reason, feel free to contribute it.

**Pull request merged**